### ISSUE TITLE
Blocks placement issue in campfire

### ISSUE THREAD

**Issue by hotspuraz** [PluginBugs/Issues-ItemsAdder#2865]
REDACTED

**Comment by LoneDev6** [comment]
Next time avoid reporting duplication bugs publicly, a new violation will lead to a termination of your account.
People can abuse duplication bugs if they are published publicly online.
I explicitly wrote that you must not report this kind of bugs on Github. 


![image](https://github.com/PluginBugs/Issues-ItemsAdder/assets/27242001/a59e99cd-e272-4a18-ab0b-5f9ced9b291f)

**Comment by LoneDev6** [comment]
I can't trigger the duplication issue, it probably is another plugin interfering.
Disable the campfire setting in config.yml of ItemsAdder if you're not interested into using it or if you don't want to find out the plugin causing this issue.

Anyway I can trigger another issue, items are put two times in case they are blocks so 1 item gets lost.
I'll fix this.

https://github.com/PluginBugs/Issues-ItemsAdder/assets/27242001/746c0440-5781-44df-ae9c-52dff619f44c

**Comment by tygh89071388** [comment]
ViaVersion 
config.yml  armor-toggle-fix: true 
To prevent this error from occurring, please change it to 'false'

**Comment by LoneDev6** [comment]
> ViaVersion config.yml armor-toggle-fix: true To prevent this error from occurring, please change it to 'false'

Thanks for the suggestion, can you please add more details about this?
Did you find this information online or in a guide?

**Comment by tygh89071388** [comment]
> > ViaVersion config.yml armor-toggle-fix: true To prevent this error from occurring, please change it to 'false'
> 
> Thanks for the suggestion, can you please add more details about this? Did you find this information online or in a guide?

It appears that my server has also experienced item duplication issues recently. Among them, there was duplication caused by being able to place certain items in the campfire. Initially, I suspected it might be related to the ItemsAdder plugin, but after numerous attempts, I discovered that the success of the duplication depends on the players' versions. Further testing revealed that the 'armor-toggle-fix' option in ViaVersion is causing this problem.
